VideoLocker/res/layout/row_course_list.xml

```diff
@@ -62,22 +62,18 @@
         android:layout_width="match_parent"
         android:layout_height="@dimen/course_detail_card_height"
         android:layout_alignParentBottom="true"
-        android:background="@color/transparent_white_85">
+        android:background="@color/transparent_white_85"
+        android:paddingEnd="10dp"
+        android:paddingLeft="10dp"
+        android:paddingRight="10dp"
+        android:paddingStart="10dp"
+        android:paddingTop="10dp"
```

Comment:
AS we're converting the child margins to parent padding, we need to ensure that all the child margins are removed (e.g. the right/end margin in `starting_from`, `new_course_content_layout`).